src/wp-includes/class-wp-customize-nav-menus.php

```diff
@@ -562,10 +562,12 @@ public function enqueue_scripts() {
 			'under'                   => __( 'Under %s' ),
 			/* translators: %s: Previous item name. */
 			'outFrom'                 => __( 'Out from under %s' ),
-			/* translators: 1: Item name, 2: Item position, 3: Total number of items. */
-			'menuFocus'               => __( '%1$s. Menu item %2$d of %3$d.' ),
-			/* translators: 1: Item name, 2: Item position, 3: Parent item name. */
-			'subMenuFocus'            => __( '%1$s. Sub item number %2$d under %3$s.' ),
+			/* translators: 1: Item name, 2: Item type, 3: Item index, 4: Total Items */
+			'menuFocus'               => __( 'Edit %1$s (%2$s, %3$d of %4$d)' ),
+			/* translators: 1: Item name, 2: Item type, 3: Item index, 4: Total Items, 5: Item Parent. */
+			'subMenuFocus'            => __( 'Edit %1$s (%2$s, sub-item %3$d of %4$d of %5$s)' ),
+			/* translators: 1: Item name, 2: Item type, 3: Item index, 4: Total Items, 5: Item Parent, 6: Item Depth */
+			'subMenuMoreDepthFocus'   => __( 'Edit %1$s (%2$s, sub-item %3$d of %4$d of %5$s, level %6$d)' ),
```

Comment:
Change to `under %5$s`. Better English grammar, makes the string easier to parse while placeholders are present.